packages/e2e-tests/scripts/save-examples.ts

```diff
@@ -432,8 +432,10 @@ async function saveBrowserExample({ example }: TestRunCallbackArgs) {
 
   // Shouldn't be "node" by this point
   await recordPlaywright((argv.runtime || example.runtime) as BrowserName, async (page, expect) => {
-    await page.goto(exampleUrl);
-    await playwrightScript(page, expect);
+    const waitForLogPromise = playwrightScript(page, expect);
```

Comment:
We need to start listening _before_ loading the page. Otherwise we'll usually miss any logs that weren't put behind a `setTimeout` (and even some of them).